### DISCUSSION TITLE
I have xvg file obtained from gromacs for many lambda states. Now I want to compute free energy of solvation using MBAR approach. I have used estimator .py file i.e, mbar_.py but couldn't able to run it. Below is the error.

### DISCUSSION THREAD

**Discussion by kishantchem** [alchemistry/alchemlyb#334] [Q&A] [answered]
Traceback (most recent call last):
  File "/home/kishant/gromacs_run/iran/drug/corrected-lambda/mbar_.py", line 5, in <module>
    from .base import _EstimatorMixOut
ImportError: attempted relative import with no known parent package

**Comment by dotsdl** [comment, marked as answer]
It appears that you pulled the `mbar_.py` module out of the source tree and then tried to run it as a script. That is a library module, not a script.

To parse files, including XVGs produced via Groamcs, I recommend reviewing the documentation: https://alchemlyb.readthedocs.io/en/latest/parsing.html

**Reply by kishantchem** [reply]
I am a user of "alchemical analysis" but the doc says that the use of this package is not encouraged. Therefore, I would like to use the alchemlyb and for that I need your help. I have seen the alchemtest and completed the benzene example. But for our case, I would like to read all xvg files for my case study.  What do we need to do instead of below two lines?

from alchemtest.gmx import load_benzene
bz= load_benzene.data
